src/components/views/voip/VideoView.js

```diff
@@ -50,7 +50,10 @@ module.exports = React.createClass({
     },
 
     getRemoteAudioElement: function() {
-        return this.refs.remoteAudio;
+        // this needs to be somewhere at the top of the DOM which
+        // always exists to avoid audio interruptions.
+        // Might as well just use DOM.
+        return document.getElementById("remoteAudio");
```

Comment:
```
sierra:vector-web matthew$ git status
On branch develop
Your branch is up-to-date with 'origin/develop'.
nothing to commit, working directory clean
sierra:vector-web matthew$ cat vector/index.html | grep remoteAudio
    <audio id="remoteAudio"/>
```

I agree it's nasty to refer to something at such a distance, but I'm not seeing a saner easier way (other than possibly passing the ref down via React contexts or something?).  For this very special case, i think DOM gutwrenching is still the least magical way to go...